VERSION

```diff
@@ -1 +1 @@
-1.11.1
+1.11.2
```

Comment:
the bump commit should always be the top commit for the release that gets tagged so as long as that happens and its top commit do as you wish.